web-filter-base/src/com/untangle/app/web_filter/WebFilterBase.java

```diff
@@ -981,6 +981,7 @@ protected void postInit()
             addCategory(categories, 85, "Self Harm", "Sensitive", "Graphic content, details around methods of harm, and content that glorifies or promotes suicide, anorexia, bulimia, and other types of self-harm.", true, true, false);
             addCategory(categories, 86, "DNS Over HTTPS", "Security", "DNS Over HTTPS (DoH) is a network protocol used to communicate domain name server information in an encrypted form over HTTPS, thus hiding the DNS query. Includes URLs of domains performing secure DNS resolution, including DNS Over HTTPS (DoH), DNS Over TLS (DoT), and other secure DNS-related domains and IP addresses.", true, true, false);
             addCategory(categories, 87, "Low-THC Cannabis Products", "Sensitive", "Sites with content containing low-THC, non-psychoactive products, including CBD oils, resin, extracts, herbs, capsules, supplements, foods, drinks, and toiletries/skin care products. This category may also include other regulated non-psychoactive substances. ", true, false, false);
+            addCategory(categories, 88, "Generative AI", "Productivity", "This category includes Artificial Intelligence tools and systems used to generate new text, images, video, audio, code, or other types of synthetic data. ", false, false, false);
```

Comment:
Since we are updating from the previous version where we added the new ids 85, 86, 87, this will add those again.  Just remove the addCategory calls for those two ids.

Reply:
@cblaise-untangle  we have 3 id's here so all three right ?

Reply:
made the needed changes